pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTenantRestletResource.java

```diff
@@ -260,7 +276,7 @@ private String getTablesServedFromTenant(String tenantName) {
     return resourceGetRet.toString();
   }
 
-  private String toggleTenantState(String tenantName, String stateStr, @Nullable String tenantType) {
+  private SuccessResponse toggleTenantState(String tenantName, String stateStr, @Nullable String tenantType) {
```

Comment:
toggle means switch between 2 states. But the StateType enum has 3 states. We should use changeTenantState to reduce the confusion -- unless you can explain in javadoc why some state is not used.

Reply:
Now we are only using 2 states here - enable / disable. So toggle still makes sense. DROP is taken care of deleteTenant API which was already there.